code/game/objects/structures/false_walls.dm

```diff
@@ -11,7 +11,7 @@
 * around us, then checks the difference.
 */
 /proc/getOPressureDifferential(var/turf/loc)
-	var/minp=16777216;
+	var/minp=16777216; //What is even the significance of this number?
```

Comment:
I was going to say it's the highest possible pressure, but then I realized I've seen higher
In other news, this function will break if the lowest-pressure adjacent tile is at a higher pressure than that number